pkgs/tools/admin/google-cloud-sdk/default.nix

```
@@ -21,6 +21,12 @@ let
   sources = system:
     data.googleCloudSdkPkgs.${system} or (throw "Unsupported system: ${system}");
 
+  components = callPackage ./components.nix {
+    snapshotPath = builtins.fetchurl data.googleCloudSdkPkgs.components;
```

```suggestion
    snapshotPath = builtins.fetchurl data.googleCloudSdkPkgs.components;
```
Move the components JSON into this repo. You'll find that Hydra doesn't allow to use `builtins.fetchurl` or IFD.